lib/ast/node.rb

```diff
@@ -42,8 +42,18 @@ class Node
 
     # Returns the children of this node.
     # The returned value is frozen.
+    # This is very useful in order to decompose nodes
```

Comment:
A better phrasing would be "The `to_a` alias is useful for decomposing nodes concisely. For example:"